ANS_JS_template.tex

```diff
@@ -93,92 +93,92 @@
 
-\subsection{Nuclear Renewable Hybrid Energy Systems}
+ \begin{subfigures}
+\begin{figure*}[h!]
+\centering
+\includegraphics[width=.75\textwidth]{TraditionalHES.PNG}
+\caption{\small \sl Traditional HES are smaller systems meant to provide power at the location where the electricity is made.  They have traditionally balanced fluctuating sources of power with reliable sources of electricity.}
+\end{figure*}
 
-Nuclear Renewable Hybrid Energy Systems (NRHES) are defined by Bragg-Sitton et al. (2014) as the "tighter coupling of nuclear and renewable energy sources in a manner that better optimizes energy use for the combined electricity, industrial manufacturing, and transportation sectors capable of apportioning thermal electrical energy to first meet the grid demand (with appropriate power conversion systems), then utilizing excess thermal and, in some cases, electrical energy to drive a process that results in an additional product \cite {Bragg-Sitton2014}".  The potential benefit of co-locating the various facilities in the NRHES is optimizing the system to minimize cost to the NRHES as a whole while increasing economic resilience by diversifying both the means of generating energy as well as the products produced. For example, if natural gas prices increase, there could be less reliance on the natural gas plant and more focus on storage.  On a more short term perspective, if electricity prices are negative, the energy produced by the NRHES can be diverted to the industrial process. The drawbacks of co-locating the various generation sources and the industrial processes are the increased complexity of the system.  Furthermore, a nuclear power plant has never been thermally coupled in the United States at this point.  The low level state of development as well as the lack of experimental data suggest a lot of rigorous technical work needing to be developed.  There are regulatory concerns with a thermally coupled NRHES  
+\begin{figure*}[h!]
+\centering
+\includegraphics[width=.75\textwidth]{MIMO_HES.PNG}
+\caption{\small \sl Figures 1a and 1b compare the MISO and MIMO configurations demonstrating the differences between traditional hybrid energy systems that are focused on generating reliable electricity and non-traditional hybrid energy systems which have the added objective of generating an additional product.  While many of the elements are the same, the MIMO system includes an industrial process which is either thermally or electrically coupled.}
+\end{figure*}
+\end{subfigures}
```

Comment:
Reviewer #3 Comment 1: Added two stacked figures of MISO and MIMO systems.
Moved the Nuclear Renewable Hybrid Energy System description section to the NRHES section.